ql/src/java/org/apache/hadoop/hive/ql/io/parquet/write/DataWritableWriter.java

```diff
@@ -536,7 +542,8 @@ public void write(Object value) {
         Long int64value = ParquetTimestampUtils.getInt64(ts, timeUnit);
         recordConsumer.addLong(int64value);
```

Comment:
The fact that we do not perform/control legacy conversion when we store timestamps in INT64 type can create problems if we end up comparing timestamps stored as INT96 and INT64. Shall we try to make the new property (`hive.parquet.timestamp.write.legacy.conversion.enabled`) affect also the INT64 storage type?

Reply:
You're right, just ignore my previous comment.